docs/pages/management/security/general-recommendations.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: General Security Recommendations
+description: Provides a simplified checklist of security measures to consider to protect enrolled resources and Teleport clusters.
+---
+
+If you host the Teleport cluster on your own network, review the following checklist for actions you can take to make 
```

Comment:
```suggestion
If you run a self-hosted Teleport cluster, review the following checklist for actions you can take to make 
```
We use the "self-hosted" language throughout the docs, so I think this is a little more straightforward.